theano/sandbox/cuda/tests/test_fftconv.py

```diff
@@ -265,16 +266,19 @@ def test_opt_conv3d_fft(self):
         assert sum(isinstance(n.op, theano.sandbox.cuda.fftconv.CuFFTOp)
                    for n in topo) == 2
 
-
         res_ref = f_ref()
         res_fft = f_fft()
 
         utt.assert_allclose(res_ref, res_fft)
 
     def test_opt_convgrad3d_fft(self):
-        inputs_shape = (16, 20, 32, 16, 1)
-        filters_shape = (10, 6, 12, 4, 1)
-        dCdH_shape = (16, 15, 21, 13, 10)
+        inputs_shape = (2, 17, 15, 16, 1)
+        filters_shape = (10, 3, 7, 4, 1)
```

Comment:
I would change the 3 to a 6 so that the computed shape for dim[1] below becomes 12 which is not a repeat of any other.